src/app/child-dev-project/attendance/dashboard-widgets/attendance-week-dashboard/attendance-week-dashboard.component.ts

```diff
@@ -47,15 +43,13 @@ export class AttendanceWeekDashboardComponent
     }
   }
 
-  ngOnInit() {
-    this.loadAttendanceOfAbsentees(this.daysOffset);
+  async ngOnInit() {
+    await this.loadAttendanceOfAbsentees(this.daysOffset);
```

Comment:
you really dont like promises

Reply:
it's showing warnings otherwise ;-)
and for unit tests this actually comes in handy to be able to await the ngOnInit call's completion